pom.xml

```diff
@@ -315,6 +315,34 @@
             </plugin>
             <!-- end source & javadoc -->
 
+            <!-- SBOM generation -->
+            <plugin>
+                <groupId>org.cyclonedx</groupId>
+                <artifactId>cyclonedx-maven-plugin</artifactId>
+                <version>2.7.3</version>
+                <configuration>
+                    <projectType>library</projectType>
+                    <schemaVersion>1.3</schemaVersion>
+                    <includeBomSerialNumber>true</includeBomSerialNumber>
+                    <includeCompileScope>true</includeCompileScope>
+                    <includeProvidedScope>true</includeProvidedScope>
+                    <includeRuntimeScope>true</includeRuntimeScope>
+                    <includeSystemScope>true</includeSystemScope>
+                    <includeTestScope>false</includeTestScope>
+                    <includeLicenseText>false</includeLicenseText>
+                    <outputFormat>all</outputFormat>
```

Comment:
This is a minor thing. Do we need to define all of these configurations ? 🤔  Seems all of them are default values

Reply:
I always go back and forth on weather to specify defaults for things like this because it's kinda self-documenting. I can go either way.

Reply:
I think it's okay to have the configurations like this as it clearly shows configuration options